src/vision/status_quo/alan_has_an_event_loop.md

```diff
@@ -0,0 +1,104 @@
+# 😱 Status quo stories: Alan has an external event loop and wants to use futures/streams
```

Comment:
cc @rust-lang/wg-async-foundations I didn't notice at first that there is a status quo story snuck in here :)